blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/PipelineApiTest.java

```diff
@@ -68,6 +70,20 @@ public void getFolderPipelineTest() throws IOException {
         validatePipeline(p, response);
     }
 
+    @Test
+    public void linkStartLimitTest() throws IOException, UnirestException {
+        MockFolder folder = j.createFolder("folder1");
+        Project p = folder.createProject(FreeStyleProject.class, "test1");
+
+        HttpResponse<String> response = Unirest.get(getBaseUrl("/organizations/jenkins/pipelines/folder1/pipelines/"))
```

Comment:
Perhaps a better test would actually specify values for `start` and `limit` here?